app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Users
+      class SessionsController < Devise::SessionsController
+        include ValidClientConcern
+
+        respond_to :json
+
+        before_action :ensure_valid_client
+
+        def create
+          user = User.find_by(email: params[:user][:email])
+          valid_provider = user&.provider == User.providers[:email]
+          valid_password = user&.valid_password?(params[:user][:password])
+
+          # TODO: Check if user has verified e-mail (#7)
+          if valid_provider && valid_password
+            render_success(user)
```

Comment:
Note to self: Add `sign_in(user)`, otherwise `current_user` might return `nil` later on